submissions/lalalilulu/hooli-style_popup/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hooli-style Popup</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav>
+        <ul class="menu">
+            <li><a href="#" class="nav-text">Mail</a></li>
+            <li><a href="#" class="nav-text">Images</a></li>
+            <li><div class="popup">
+                <img class="popup-icon" src="icons/popup-button.png" alt="popup">
```

Comment:
Remove menu elements shifting on hover, it's bad for page performance. [Watch video](https://share.vidyard.com/watch/Cmwa1XM1CmXbcxK2Xtz8ba?)

Reply:
Not fixed yet.